common/http/src/main/java/io/helidon/common/http/Content.java

```diff
@@ -40,7 +40,9 @@
  * It is possible to register function to convert publisher to {@link CompletionStage} of a single entity using
  * {@link #registerReader(Class, Reader)} or {@link #registerReader(Predicate, Reader)} methods. It
  * is then possible to use {@link #as(Class)} method to obtain such entity.
+ * @deprecated use {@code io.helidon.media.common.MessageBodyReadableContent} instead
  */
+@Deprecated
```

Comment:
As this is a backward incompatible version - should we keep `Content` in and remove it in `3.0`, or just remove it immediately?

Reply:
That's a very good question. I did the work initially for 1.x, now that we are at 2.x I think we should rethink this. Let's track this item in a separate if you agree.

Reply:
E.g., I'd like to remove the old Reader / Writer  interfaces with all the corresponding registerReader / registerWriter method and instead use the new .context() methods.

The `.content()` method remains, so that keeps some kind of compatibility, also I'm positive that there is not much code declaring variables of type `Content` ; so we should feel OK in removing the `Content` interface.

Reply:
I've created https://github.com/oracle/helidon/issues/1375